### ISSUE TITLE
Model mapping

### ISSUE THREAD

**Issue by Dennise89** [WendellAdriel/laravel-validated-dto#25]
Hello,

Is it possible to map data to the model? There is a nice function `->toModel()`, but my model's fields are named different.

In my request I have a data array with the field in it. For example:`'attribute' => 'value', data => ['test' => 'testvalue']` But in my model it's just simply called test, but the attribute also exists in my model. So it's filling the attribute data, but not the test attribute.

**Comment by smartmeerkat** [comment]
Hi, @Dennise89! I already made this logic and make PR soon.

**Comment by Dennise89** [comment]
> Hi, @Dennise89! I already made this logic and make PR soon.

Both features?

**Comment by smartmeerkat** [comment]
> 

Yes. There will be pipelines where you can customize data before validation or before export.

**Comment by smartmeerkat** [comment]
@Dennise89 [Feel free to discuss and extend.](https://github.com/WendellAdriel/laravel-validated-dto/pull/26)

**Comment by WendellAdriel** [comment]
Hey @Dennise89 this is not yet available but it's a great idea!
Thanks for the contribution

**Comment by WendellAdriel** [comment]
@smartmeerkat I'm going to check the PR

**Comment by WendellAdriel** [comment]
@Dennise89 @smartmeerkat I had some time to work on this and created this PR: #29
